pkg/kudoctl/cmd/get/get_test.go

```diff
@@ -61,7 +61,7 @@ func TestGetInstances(t *testing.T) {
 		},
 	}
 
-	testOperator := &v1beta1.Operator{
+	testOperator := &kudoapi.Operator{
```

Comment:
isn't there a separate PR for this?   can we separate this out?

Reply:
Yes, this shouldn't have been in here.